misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/ExecutionContext.java

```diff
@@ -16,27 +16,36 @@
 
 package org.ballerinalang.debugadapter;
 
-import com.sun.jdi.VirtualMachine;
+import com.sun.jdi.request.EventRequestManager;
 import io.ballerina.projects.Project;
+import org.ballerinalang.debugadapter.jdi.VirtualMachineProxyImpl;
 import org.eclipse.lsp4j.debug.services.IDebugProtocolClient;
 
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.Optional;
+
 /**
  * Debugging session context.
  */
-public class DebugContext {
+public class ExecutionContext {
```

Comment:
Any specific reasons for changing the class name here?
If there is a valid reason shall we update the doc comment accordingly?

Reply:
yeah thought of using a more specific name, since we are already having a separate context holder for debugging suspended state.